recipes/openpiv/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "0.20" %}
+
+package:
+  name: openpiv
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/alexlib/openpiv-python.git
```

Comment:
Also, what is the reason for using a fork? We prefer to use the original source if possible. If we require patches, those can be applied (depending on their purpose). However, those patches should be submitted upstream.